sim/druid/mangle.go

```diff
@@ -95,9 +115,9 @@ func (druid *Druid) registerMangleCatSpell() {
 			IgnoreHaste: true,
 		},
 
-		DamageMultiplier: (1 + 0.1*float64(druid.Talents.SavageFury)) * weaponMulti,
-		ThreatMultiplier: 1,
-		BonusCoefficient: 1,
+		DamageMultiplierAdditive: (1 + 0.1*float64(druid.Talents.SavageFury)) * weaponMulti,
```

Comment:
This should also be split into additive / not additive. The weapon multi is multiplicative but Savage fury addittive
```suggestion
    DamageMultiplier: 1 * weaponMulti,
		DamageMultiplierAdditive: 1 + 0.1*float64(druid.Talents.SavageFury),
```

Reply:
How does DamageMultiplierAdditive work under the hood? Since as far as I know, Savage Fury is a flat 1.2x mod on all mangle damage done.